CHANGELOG.md

```diff
@@ -23,9 +23,12 @@ This release contains several large new features. In particular, the shoulder su
 
 * During scheme parsing, folders found in the scheme folder not present in the assets (when configured) are removed
 * Shoulder surfing prevention:
-  * The `server.SessionPackage` struct now contains an extra field `FrontendAuth`
-  * The `irma.Qr` struct now contains an optional field `PairingRecommended` (named `pairingHint` when being marshalled to JSON) that is set to true when pairing is recommended for that session, as indication to the frontend
-  * The `StartSession` function from the API of the `irmaserver` package now returns three values: the session pointer (type `*irma.QR`), the requestor token (type `irma.RequestorToken`) and the frontend authorization token (type `irma.FrontendAuthorization`)
+  * The `server.SessionPackage` struct now contains a new struct `FrontendRequest` of type `*irma.FrontendSessionRequest`, containing (a.o.) the following:
+    * A boolean `PairingRecommended` (named `pairingHint` when being marshalled to JSON) that is set to true when pairing is recommended for that session, as indication to the frontend
+    * An `Authorization` token used by the frontend to set pairing options
```

Comment:
Do we also want to mention that the frontend request also mentions the supported API versions of the frontend endpoints? (with `minProtocolVersion` and `maxProtocolVersion`)?